src/ODC-By-1.0.xml

```diff
@@ -0,0 +1,504 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="ODC-By-1.0"
+            name="Open Data Commons Attribution License v1.0">
+      <crossRefs>
+         <crossRef>https://opendatacommons.org/licenses/by/1.0/</crossRef>
+      </crossRefs>
+    <text>
+      <titleText>
+	     <optional>##</optional>
+         <p>ODC Attribution License (ODC-By)</p>
+      </titleText>
+      <optional>###</optional>
+      <p>Preamble</p>
+      <p>The Open Data Commons Attribution License is a license agreement intended to allow users to freely
+      share, modify, and use this Database subject only to the attribution requirements set out in Section 4.</p>
+      <p>Databases can contain a wide variety of types of content (images, audiovisual material, and sounds
+      all in the same database, for example), and so this license only governs the rights over the Database,
+      and not the contents of the Database individually. Licensors may therefore wish to use this license
+      together with another license for the contents.</p>
+      <p>Sometimes the contents of a database, or the database itself, can be covered by other rights not
+      addressed here (such as private contracts, trademark over the name, or privacy rights / data protection
+      rights over information in the contents), and so you are advised that you may have to consult other
+      documents or clear other rights before doing activities not covered by this License.</p>
+      <p>------</p>
+    
+      <p>The Licensor (as defined below)</p>
+      <p>and</p>
+      <p>You (as defined below)</p>
+      <p>agree as follows:</p>
+      <list>
+        <item>
+            <bullet>1.0</bullet>
+          Definitions of Capitalised Words
+          <p>"Collective Database" - Means this Database in unmodified form as part of a
+             collection of independent databases in themselves that together are assembled into a
+             collective whole. A work that constitutes a Collective Database will not be considered a
+             Derivative Database.</p>
+            <p>"Convey" - As a verb, means Using the Database, a Derivative Database, or the
+             Database as part of a Collective Database in any way that enables a Person to make or receive
+             copies of the Database or a Derivative Database. Conveying does not include interaction with a
+             user through a computer network, or creating and Using a Produced Work, where no transfer of a
+             copy of the Database or a Derivative Database occurs.</p> 
+             <p>  "Contents" - The
```

Comment:
Doesn't really matter, but you may want to remove the space from `<p> "Contents"` for consistency with the other entries.